docs/docs/user/references/config.md

```diff
@@ -1,12 +1,12 @@
 # Configuration
```

Comment:
I don't like "Configuration", but I'm not sure whether "Pipeline Configuration" is better, because then shouldn't "Defaults" also be named "Pipeline Defaults"? Components, too

Reply:
I agree with that. We should add the `Pipeline configuration` and `Pipeline defaults`

Reply:
What about Components?

Reply:
hm, isn't the whole point of `config.yaml` that it contains global configuration options, not specific to some pipeline? (this all goes into `pipeline.yaml`)

Reply:
imo _Configuration_ (or _Config_ for short) is appropriate

Reply:
So you would say it configures KPOps rather than pipelines? Options like broker URL and defaults_path make it a bit specific to a pipeline IMO. Anyhow, I feel like adding "pipeline" in the titles is redundant. There isn't much to be configured in KPOps' core behavior from outside the code.

I would probably like to see improved categorization in the nav bar after the release. Right now I feel like the references section is a bit of a place for everything and so the titles of the pages are not as clear.

Reply:
well... `config.yaml` holds the configuration for kpops. but of course kpops _is_ all about pipelines. Let's not overcomplicate it :)

yeah, I agree about further optimizing the docs structure, although I feel like we're at a pretty good point now with regards to the release (thanks to your efforts)